db/schema.rb

```diff
@@ -162,9 +162,9 @@
     t.string "why_dont_you_know_la"
     t.integer "unitletas"
     t.integer "builtype"
+    t.datetime "property_void_date"
```

Comment:
Do you get this when you run migrations on a fresh DB? If not should probably revert this.